scripts/website/expand_templates/main.go

```diff
@@ -92,29 +93,35 @@ type latestRelease struct {
 }
 
 func getLatestVersion() (string, error) {
-	req, err := http.NewRequest( //nolint:noctx
-		http.MethodGet,
-		"https://api.github.com/repos/golangci/golangci-lint/releases/latest",
-		http.NoBody,
-	)
+	endpoint := "https://api.github.com/repos/golangci/golangci-lint/releases/latest"
+
+	//nolint:noctx
```

Comment:
Could we change `getLatestVersion` to accept context and remove this `//nolint:noctx`?

```go
func getLatestVersion(ctx context.Context) (string, error) {
```

We can pass at least `context.Background()` when calling `getLatestVersion`.

Reply:
it's useless to use `context.Background()` if there is nothing based on this context.

Reply:
Yeah, you are right. I think we can add a timeout, e.g. 2 seconds. Sometimes even GitHub has [issues](https://www.githubstatus.com/history).

Reply:
the timeout for an `htttp.Client` should be set on the client, not on the request.